project/Dependencies.scala

```diff
@@ -21,9 +21,9 @@ object Dependencies {
     // Scala
     val json4s          = "3.2.11"
     val scalaz7         = "7.0.9"
-    val collUtils       = "6.20.0"
```

Comment:
I'm bit hazy about such a huge leap. Chances are big that other libraries are not 18-compatible yet which means we can hit runtime exceptions. If there's a need for this bump I'd like to first check if scala-common-enrich is compatible with it.

Reply:
I won't be insisting on collection-utils. The only reason was that there was a big gap and I just wanted to catch up.
I will doing some work on `scala-common-enrich` this week as part of optional enrichment work. I can check the compatibility if you want.

Reply:
Yes, MiMa is quite wide-spread. I was thinking about it the other day, but decided that we have not that big code estate to have troubles with maintenance. On the other hand we have the clear evidence that outside contributors are not always aware of dependency tree and we have few internal issues with it, so it might be a good idea to reconsider. What do you think @BenFradet?

Reply:
yes we will have to at some point, if we can rack up some xp on a guinea pig project (i.e. this one) that'd be great.

Reply:
That's exciting to hear. Helps a lot for moving forward with the caution needed. 🙌  ✌️